src/func/alias.js

```diff
@@ -94,10 +95,12 @@ export async function updateAliasSet(
 	return newSet.save(null, {transacting});
 }
 
-export function getAliasByIds(
+export async function getAliasByIds(
 	transacting: Transaction, ids: Array<number>
 ): Promise<Object> {
-	return transacting.select('*')
+	const aliases = await transacting.select('*')
 		.from('bookbrainz.alias')
 		.whereIn('id', ids);
+	return aliases.reduce((aliasesMap, alias) =>
+		_.extend(aliasesMap, {[alias.id]: snakeToCamel(alias)}), {});
```

Comment:
Why `_.extend` and not `_.assign`?

    return aliases.reduce((aliasesMap, alias) => _.assign(aliasesMap, {[alias.id]: snakeToCamel(alias)}), {});

Or with ES6:

    return aliases.reduce((aliasesMap, alias) => ({...aliasesMap, {[alias.id]: snakeToCamel(alias)}}), {});